### PR TITLE
Badge

### PR DESCRIPTION
**Pull request by bajajcodes**
- Badge for icons on top left position only with single and multiple digits(multiple digits are considered as overlfow)
- Badge for avatars shows status, used avatar component for helper

`Note:`  Long-hand property is not prefered to use.

127 0 0 1_5500_examples_badge html (2)

### CODE REVIEW

File: css/components/badge.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
+@import "../helpers/spacing.css";
+@import "../utilities/variables.css";
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
```
Comment (Hamdrive) [inline css/components/badge.css:3]: If you are using this same font overall in your CL, then keep this import in index.css or wherever you import all your stylesheets

File: css/components/badge.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+.badge-count-overflow{
+    top:-40%;
+    left:50%; 
+}
+
+.badge-count-color-pos{
+    top:-20%;
+    left:50%; 
+}
```
Comment (Hamdrive) [inline css/components/badge.css]: Add new line here

**Review by Hamdrive — COMMENTED** [review]
Looks good overall

File: css/components/badge.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    background-color: var(--grey);
+}
+
+.count-blue{
+    background-color: var(--blue);
+}
+
+.badge-count{
+    color: var(--white);
+    width:calc(var(--size-10));
```
Comment (raghav2110) [inline css/components/badge.css]: Did not understand use of calc here.

**Review by raghav2110 — COMMENTED** [review]
Overall code looks good to me.

File: css/components/badge.css (hunk 1 above)
Comment (bajajcodes) [reply css/components/badge.css:3]: This url is for icons and I will keep this import in `index.css` from now onwards.

File: css/components/badge.css (hunk 2 above)
Comment (bajajcodes) [reply css/components/badge.css]: I have added new line there.

File: css/components/badge.css (hunk 3 above)
Comment (bajajcodes) [reply css/components/badge.css]: Hey @raghav2110 thanks for pointing out this, I missed it.

Actually I was setting width and height dynamically and put it there by mistake.

Now I have removed it from there.

File: css/components/badge.css (hunk 2 above)
Comment (bajajcodes) [reply css/components/badge.css]: @Hamdrive mentioned link

https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

**Pull request merged**